sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -81,6 +82,18 @@
    */
   public static final String FULLSCREEN_CHANGED_NOTIFICATION_NAME = "fullscreenChanged";
 
+  /**
+   * OoyalaNotification name when the VR mode has changed to MONO.
+   * No "data" is passed in the OoyalaNotification.
+   */
+  public static final String VR_MODE_MONO_NOTIFICATION_NAME = "vrModeMono";
```

Comment:
Let's create a VR_MODE_CHANGED_NOTIFICATION_NAME, and use it like
`sendNotification(VR_MODE_MONO_NOTIFICATION_NAME, "vrModeStereo");`

Reply:
Ok